### PR TITLE
Add docs about parameter transformation

### PR DESCRIPTION
**Pull request by oyvindeide**
**Issue**
Resolves #my_issue

**Approach**
_Short description of the approach_

## Pre review checklist

- [ ] Added appropriate release note label
- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

File: docs/reference/configuration/keywords.rst [hunk 1]
```diff
@@ -958,7 +958,15 @@ and/or history matching project.
         In general, the first keyword on each line in the parameter configuration file
         defines a key, which when found in the template file enclosed in ``<`` and ``>``,
         is replaced with a value. The rest of the line defines a prior distribution
-        for the key. The various prior distributions available for the ``GEN_KW``
+        for the key.
+
+        **Note that ert only stores values between 0 - 1, and the transformation is**
```
Comment (sondreso) [inline docs/reference/configuration/keywords.rst:963]:
```suggestion
        **Note that ERT only stores values sampled from a standard normal distributions, and a transformation is**
```

**Review by sondreso — APPROVED** [review]
Should we also mention that this is a technical detail, and something that we want to change?

**Comment by oyvindeide** [comment]
> Should we also mention that this is a technical detail, and something that we want to change?

Would prefer if we keep that out of the documentation, and keep it in the road map. We already have quite a few similar statements already that are very stale. Think it is better if we stick to the facts in the documentation, and add future plans elsewhere. But will add it if you would like.

**Comment by sondreso** [comment]
> Would prefer if we keep that out of the documentation, and keep it in the road map. We already have quite a few similar statements already that are very stale. Think it is better if we stick to the facts in the documentation, and add future plans elsewhere. But will add it if you would like.

I agree! Do we have an issue or will you create it, such that we can get it in to the roadmap? 🙂

**Comment by oyvindeide** [comment]
> I agree! Do we have an issue or will you create it, such that we can get it in to the roadmap? slightly_smiling_face

Added the issue here: https://github.com/equinor/ert/issues/4238 and put it in our TODO.

**Pull request merged**